src/sentry/tsdb/snuba.py

```diff
@@ -331,7 +338,9 @@ def trim(self, result, groups, keys):
                     else:
                         del result[rk]
 
-    def get_range(self, model, keys, start, end, rollup=None, environment_ids=None):
+    def get_range(
+        self, model, keys, start, end, rollup=None, environment_ids=None, snuba_filters=None
+    ):
```

Comment:
This also breaks the abstraction since `get_range` is defined on the base class.

Reply:
added this too